exporters/elasticsearch/test/es_log_exporter_test.cc

```diff
@@ -0,0 +1,64 @@
+#include "opentelemetry/exporters/elasticsearch/es_log_exporter.h"
+#include "opentelemetry/ext/http/server/http_server.h"
+#include "opentelemetry/logs/provider.h"
+#include "opentelemetry/sdk/logs/exporter.h"
+#include "opentelemetry/sdk/logs/log_record.h"
+#include "opentelemetry/sdk/logs/logger_provider.h"
+#include "opentelemetry/sdk/logs/simple_log_processor.h"
+
+#include <gtest/gtest.h>
+#include <iostream>
+
+namespace sdklogs       = opentelemetry::sdk::logs;
+namespace logs_api      = opentelemetry::logs;
+namespace nostd         = opentelemetry::nostd;
+namespace logs_exporter = opentelemetry::exporter::logs;
+
+// Attempt to write a log to an invalid host/port, test that the timeout works properly
+TEST(ElasticsearchLogsExporterTests, InvalidEndpoint)
+{
+  // Create options for the elasticsearch exporter
+  logs_exporter::ElasticsearchExporterOptions options("localhost", -1, "logs", 5, true);
+  options.response_timeout_ = 10;  // Wait 10 seconds to receive a response
```

Comment:
is this overkill 😂

Reply:
Considering Lalit's http unit test does this except with 30 seconds, I think its fine :)

Reply:
Ohh I see yeah, I get wait during the CI and I'm always afraid it's deadlocking or sth, gotcha gotcha